--- go-ipfs-config@v0.0.3/config.go	2019-06-10 11:17:00.000000000 -0700
+++ /tmp/go-ipfs-config/config.go	2019-06-10 15:55:29.000000000 -0700
@@ -33,13 +33,13 @@
 
 const (
 	// DefaultPathName is the default config dir name
-	DefaultPathName = ".ipfs"
+	DefaultPathName = ".btfs"
 	// DefaultPathRoot is the path to the default config dir location.
 	DefaultPathRoot = "~/" + DefaultPathName
 	// DefaultConfigFile is the filename of the configuration file
 	DefaultConfigFile = "config"
 	// EnvDir is the environment variable used to change the path root.
-	EnvDir = "IPFS_PATH"
+	EnvDir = "BTFS_PATH"
 )
 
 // PathRoot returns the default configuration root directory
