--- client.go	2019-08-16 11:18:19.000000000 -0700
+++ /tmp/go-ipfs-cmds/client.go	2019-08-16 11:33:19.000000000 -0700
@@ -93,7 +93,7 @@
 				// XXX: this runs the PreRun twice
 				return c.fallback.Execute(req, re, env)
 			}
-			err = fmt.Errorf("cannot connect to the api. Is the daemon running? To run as a standalone CLI command remove the api file in `$IPFS_PATH/api`")
+			err = fmt.Errorf("cannot connect to the api. Is the daemon running? To run as a standalone CLI command remove the api file in `$BTFS_PATH/api`")
 		}
 		return err
 	}
