--- ipfs/interface-go-ipfs-core@v0.0.6/path.go	2019-06-07 14:48:23.000000000 -0700
+++ /Users/tron/patch-from/interface-go-ipfs-core/v0.0.6/path.go	2019-06-07 15:07:37.000000000 -0700
@@ -114,7 +114,7 @@
 // IpfsPath creates new /ipfs path from the provided CID
 func IpfsPath(c cid.Cid) ResolvedPath {
 	return &resolvedPath{
-		path:      path{ipfspath.Path("/ipfs/" + c.String())},
+		path:      path{ipfspath.Path("/btfs/" + c.String())},
 		cid:       c,
 		root:      c,
 		remainder: "",
@@ -166,7 +166,7 @@
 
 func (p *path) Mutable() bool {
 	//TODO: MFS: check for /local
-	return p.Namespace() == "ipns"
+	return p.Namespace() == "btns"
 }
 
 func (p *resolvedPath) Cid() cid.Cid {
